src/actions.js

```diff
@@ -29,36 +29,39 @@ export const CLOSE_TILE = "CLOSE_TILE";
 
 // TimeseriesActions
 export const ADD_TIMESERIES = "ADD_TIMESERIES";
+export const FETCH_TIMESERIES_EVENTS = "FETCH_TIMESERIES_EVENTS";
+export const RECEIVE_TIMESERIES_EVENTS = "RECEIVE_TIMESERIES_EVENTS";
+export const FETCH_RASTER_EVENTS = "FETCH_RASTER_EVENTS";
+export const RECEIVE_RASTER_EVENTS = "RECEIVE_RASTER_EVENTS";
 
-// Alarms
-export const fetchAlarmsAction = () => {
-  return {
-    type: FETCH_ALARMS
-  };
-};
-
-export const receiveAlarmsAction = alarms => {
+export const receiveAlarmsAction = (alarms, isTimeseries) => {
```

Comment:
This isn't wrong but I would prefer functions like this to not have the word 'action' in the name.